rites/roman1969/src/catalog/martyrology.ts

```diff
@@ -1216,6 +1222,9 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
     dedication_of_the_cathedral_of_saint_stephen_of_toulouse_france: {
       dateOfDedication: '1592-08-30',
     },
+    dedication_of_the_cathedral_of_the_immaculate_conception_hong_kong: {
+      name: 'Dedication of the Cathedral of the Immaculate Conception, Hong Kong', // src: http://catholic-dlc.org.hk/1209_Dedication_of_the_Cathedral-enA5.pdf
+    },
```

Comment:
In this case, I made a mistake myself by placing the source comment after the `name` property, while all sources in `martyrology.ts` are usually placed before the cited data.

```suggestion
    // src: http://catholic-dlc.org.hk/1209_Dedication_of_the_Cathedral-enA5.pdf
    dedication_of_the_cathedral_of_the_immaculate_conception_hong_kong: {
      name: 'Dedication of the Cathedral of the Immaculate Conception, Hong Kong',
    },
```